deps/wrapper.cc

```diff
@@ -115,14 +119,45 @@ std::string ToString(ray::FunctionDescriptor function_descriptor)
     return function_descriptor->ToString();
 }
 
+ObjectID submit_task() {
+    auto &worker = CoreWorkerProcess::GetCoreWorker();
```

Comment:
I assume this is the driver process?

Reply:
I understand this to be the client side interface to Ray for both workers and drivers. It's not a reference to the driver process but a connection to the Raylet running on the node. In this particular case it is the driver core worker connection we establish in `initialize_coreworker`.

Reply:
that's my understanding as well.  the "core worker" is the client API that's used by BOTH the driver and the workers